Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm

```diff
@@ -1423,6 +1423,10 @@ static RecommendDesktopClassBrowsingForRequest desktopClassBrowsingRecommendedFo
     if (equalLettersIgnoringASCIICase(host, "roblox.com"_s) || host.endsWithIgnoringASCIICase(".roblox.com"_s))
         return RecommendDesktopClassBrowsingForRequest::No;
 
+    // FIXME: Remove this quirk when <rdar://122481999> is complete
+    if (equalLettersIgnoringASCIICase(host, "spotify.com"_s) || host.endsWithIgnoringASCIICase(".spotify.com"_s))
```

Comment:
@brentfulgham 

```suggestion
    if (equalLettersIgnoringASCIICase(host, "spotify.com"_s) || host.endsWithIgnoringASCIICase(".spotify.com"_s) || equalLettersIgnoringASCIICase(host, "open.spotifycdn.com"_s))
```

All the JS are on `open.spotifycdn.com`.

Reply:
Is `open.spotifycdn.com` the host of the top URL?